### PR TITLE
optimize: optimize ColumnUtils.addEscape method performance

### PR DESCRIPTION
**Pull request by yangfuhai**
### Ⅰ. Describe what this PR did

optimize ColumnUtils.addEscape method performance

### CODE REVIEW

File: rm-datasource/src/main/java/io/seata/rm/datasource/ColumnUtils.java [hunk 1]
```diff
@@ -156,7 +156,16 @@ public static String addEscape(String colName, Escape escape) {
...
             return colName;
         }
-        return String.format("%s%s%s", escape.value, colName, escape.value);
+
+        char[] buf = new char[colName.length() + 2];
+        buf[0] = escape.value;
+        buf[buf.length - 1] = escape.value;
+
+        for (int i = 0; i < colName.length(); i++) {
+            buf[i + 1] = colName.charAt(i);
```
Comment (slievrly) [inline rm-datasource/src/main/java/io/seata/rm/datasource/ColumnUtils.java]: System.arrayCopy

File: rm-datasource/src/main/java/io/seata/rm/datasource/ColumnUtils.java [hunk 2]
```diff
@@ -156,7 +156,16 @@ public static String addEscape(String colName, Escape escape) {
...
+        }
+
+        return new String(buf);
```
Comment (slievrly) [inline rm-datasource/src/main/java/io/seata/rm/datasource/ColumnUtils.java]: intern

File: rm-datasource/src/main/java/io/seata/rm/datasource/ColumnUtils.java (hunk 1 above)
Comment (yangfuhai) [reply rm-datasource/src/main/java/io/seata/rm/datasource/ColumnUtils.java]: use String.getChars() replace System.arraycopy()

**Comment by xingfudeshi** [comment]
Please resolve conflicting file.

**Comment by l81893521** [comment]
Please solve the conflict.

**Pull request merged**